zebra-state/src/service/non_finalized_state/chain/index.rs

```diff
@@ -39,9 +42,10 @@ pub struct TransparentTransfers {
     /// The `getaddressutxos` RPC doesn't need these transaction IDs to be sorted in chain order,
     /// but it might in future. So Zebra does it anyway.
     ///
-    /// TODO:
-    /// - to avoid [`ReadStateService`] response inconsistencies when a block has just been finalized,
-    ///   combine the created UTXOs, combine the spent UTXOs, and then remove spent from created
+    /// TODO: To avoid [`ReadStateService`](crate::service::ReadStateService)
+    /// response inconsistencies when a block has just been finalized, combine
+    /// the created UTXOs, combine the spent UTXOs, and then remove spent from
+    /// created
     ///
     /// Optional:
```

Comment:
This has already been done

```suggestion
    /// Optional TODOs:
```